static/app/components/group/suggestedOwnerHovercard.tsx

```diff
@@ -62,6 +63,18 @@ function SuggestedOwnerHovercard(props: Props) {
     source: 'suggested_assignees',
   };
 
+  const tagColors = useMemo(
```

Comment:
memoized because the previous object was static, but this is likely not required